warehouse/forklift/legacy.py

```diff
@@ -870,8 +871,6 @@ def file_upload(request):
             for chunk in iter(
                     lambda: request.POST["content"].file.read(8096), b""):
                 file_size += len(chunk)
-                if file_size > file_size_limit:
```

Comment:
I originally moved it because the test was failing with an error about file size, which I naively assumed was due to an order of operations thing. But now that I'm looking at it, the test fails because the size of the bytes buffer used is `legacy.MAX_FILESIZE + 1`. I'm assuming that was copied and pasted from the previous test (that checks file size). I'll shore up the test and move things back.